Vagrantfile

```diff
@@ -6,72 +6,21 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
```

Comment:
@seanlip That's a "fancy" way of making a file in Linux, normally you pipe that into `cat` like this

```bash
cat to_lazy_to_open_editor << STOP_VAR # until I say "stop" consider what follows a file (single string)
This is some very imaginative text
Blah, blah, blah
STOP_VAR # End of file
```

Usually people use `EOF` (end of file) as "STOP_VAR" out of convention.

If you really want to know the details, here's a [handy SO discussion](http://stackoverflow.com/questions/2500436/how-does-cat-eof-work-in-bash) about the so called "Here-Files" (That would be the term you need to google :wink:)